doc/README.md

```diff
@@ -8,38 +8,23 @@ formatting. You do not need to learn the entire style guide before contributing
 to documentation. Someone can always edit your material later to conform with
 this guide.
 
-* Documentation is in markdown files with names formatted as
-  `lowercase-with-dashes.md`.
-  * Use an underscore in the filename only if the underscore is part of the
-    topic name (e.g., `child_process`).
-  * Some files, such as top-level markdown files, are exceptions.
-* Documents should be word-wrapped at 80 characters.
-* `.editorconfig` describes the preferred formatting.
-  * A [plugin][] is available for some editors to apply these rules.
-* Check changes to documentation with `make test-doc -j` or `vcbuild test-doc`.
-* [Use US spelling][].
-* [Use serial commas][].
-* Avoid first-person pronouns (_I_, _we_).
-  * Exception: _we recommend foo_ is preferable to _foo is recommended_.
-* Use gender-neutral pronouns and gender-neutral plural nouns.
-  * OK: _they_, _their_, _them_, _folks_, _people_, _developers_
-  * NOT OK: _his_, _hers_, _him_, _her_, _guys_, _dudes_
-* When combining wrapping elements (parentheses and quotes), place terminal
-  punctuation:
-  * Inside the wrapping element if the wrapping element contains a complete
-    clause.
-  * Outside of the wrapping element if the wrapping element contains only a
-    fragment of a clause.
-* Documents must start with a level-one heading.
+## Formatting and Structure
+
+### Headings
+
+* Each page must have a single `#`-level title at the top.
+* Chapters in the same page must have `##`-level headings.
+* Sub-chapters need to increase the number of `#` in the heading according to
+  their nesting depth.
+* The page's title must follow [APA title case][title-case].
```

Comment:
I don't think we should cite APA here or anywhere else. It muddies the waters. We already have a style guide.

Reply:
I was wondering if it makes sense to avoid mixing APA title case and Microsoft's sentence case and just stick to one for all titles because most of the documentation already adheres to the Microsoft style guide. Is it a requirement of https://github.com/electron/docs-parser?